Lombiq.ContentEditors/Assets/Scripts/async-editor/async-editor.js

```diff
@@ -120,13 +120,24 @@ window.asyncEditor.editor = {
         },
         submitEditor(nextEditorGroup) {
             const self = this;
-
-            self.api.submitEditor(
-                self.contentId,
-                self.editorGroup,
-                nextEditorGroup,
-                new FormData(self.$refs.editorForm),
-                (success, data) => { self.processApiData(success, data); });
+            const submittingEditorEvent = new CustomEvent('asyncEditorSubmittingEditor', {
+                bubbles: true,
+                cancelable: true,
+                detail: { asyncEditor: window.asyncEditor }
+            });
+
+            var form = document.querySelector('form.asyncEditor__form');

```

Comment:
I think it should be either the current async editor element (you can acquire it from the Vue object somehow) or put it to the document object. This way it'll be fired from all async editors if there are multiple on one page.